hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/SerializationConstants.java

```diff
@@ -127,10 +127,12 @@
 
     public static final int JAVA_DEFAULT_TYPE_PRIORITY_QUEUE = -49;
 
+    public static final int JAVA_DEFAULT_TYPE_OPTIONAL = -50;
```

Comment:
Is it OK to continue in row of negative ids?

Reply:
Yes, it is ok.